### ISSUE TITLE
Minecraft crashes when Reaping is with Haema

### ISSUE THREAD

**Issue by sschr15** [Jamalam360/ReapingFabric#7]
[ReapingHelper](https://github.com/JamCoreModding/ReapingFabric/blob/6f936741e03effa43e3248eb4a508198bf46743e/src/main/java/io/github/jamalam360/reaping/ReapingHelper.java#L70) runs some code which causes Haema to crash due to NPE. If a null check is added here or there, the issue would likely be solved.

**Comment by Jamalam360** [comment]
Thanks, what's actually null/could you send the error? My computer refuses to run the CF launcher at the moment so I can't test it myself without manually installing

**Comment by sschr15** [comment]
```
java.lang.NullPointerException: Cannot invoke "LivingEntity.getMainHandStack()" because "attacker" is null
	at net.minecraft.entity.damage.DamageSource.handler$bkj000$addVampireEffectiveSources(DamageSource:1027)
	at net.minecraft.entity.damage.DamageSource.player(DamageSource:57)
	at io.github.jamalam360.reaping.ReapingHelper.tryReap(ReapingHelper.java:70)
```
`toolStack.getHolder()` ends up null

**Comment by Jamalam360** [comment]
Thanks

**Comment by sschr15** [comment]
You could also run a call to `ItemStack.setHolder` which would likely fix the issue

**Comment by Jamalam360** [comment]
Thanks for reporting this, should be fixed in 1.6.1
